core/templates/core/community-update.html

```diff
@@ -26,15 +26,144 @@ <h3 class="h3">Update Community: {{ community.name }}</h3>
+                        <div class="form-group mb-4">
+                            {% if field.name == "avatar" or field.name == "background" %}
+                            <label class="form-label">{{ field.label }}</label>
+                            {% if field.value %}
+                            <div class="form-check mb-2">
+                                <input type="checkbox" name="{{ field.html_name }}-clear" id="{{ field.auto_id }}_clear"
+                                    class="form-check-input">
+                                <label class="form-check-label text-danger" for="{{ field.auto_id }}_clear">
+                                    Clear {{ field.label }}
+                                </label>
+                            </div>
+                            {% endif %}
+
+                            <input type="{{ field.field.widget.input_type }}" name="{{ field.html_name }}"
+                                id="{{ field.auto_id }}" class="form-control">
+                            {% if field.help_text %}
+                            <small class="form-text">{{ field.help_text }}</small>
+                            {% endif %}
+                            {% if field.errors %}
+                            {% for err in field.errors %}
+                            <div class="alert alert-danger" role="alert">
+                                <p class="form-error">{{ err }}</p>
+                            </div>
+                            {% endfor %}
+                            {% endif %}
+                            {% else %}
                             {{ field|as_crispy_field }}
+                            {% endif %}
                         </div>
                         {% endfor %}
-                        <button class="btn btn-sm btn-success" type="submit">Update Community</button>
+
+                        <button class="btn btn-success" type="button" onclick="confirmSubmit()">Update
+                            Community</button>
                     </form>
                 </div>
             </div>
         </div>
+        <div class="col-lg-6">
+            <div class="card bg-dark text-light mb-4" id="previewCard">
+                <div class="card-header position-relative p-0" style="height: 200px; overflow: visible;">
+                    <img id="previewBackground" src="
+                      {% if form.instance.background %}
+                        {{ form.instance.background.url }}
+                      {% else %}
+                        /media/community_backgrounds/default.jpg
+                      {% endif %}
+                    " alt="Banner preview" class="w-100 h-100" style="object-fit: cover;">
+
+                    <img id="previewAvatar" src="
+                      {% if form.instance.avatar %}
+                        {{ form.instance.avatar.url }}
+                      {% else %}
+                        /media/community_avatars/default.jpg
```

Comment:
Another node, just for future: hardcoding is bad idea. I would probably put it in settings so can manage in one place. But, given model has avatar_url  attribute, it should be switched to model.avatar_url and should work.